Source/Thunder/PluginServer.h

```diff
@@ -2000,7 +2000,7 @@ namespace PluginHost {
             private:
                 using Observers = std::vector<IShell::ICOMLink::INotification*>;
                 using Proxy = std::pair<uint32_t, const Core::IUnknown*>;
-                using Proxies = std::vector<Proxy>;
+                using Proxies = std::list<Proxy>;
```

Comment:
Why are we revrting back to the std::list std::vector is faster and cheaper here, so is there a real need for std::list?